src/Mod/TechDraw/Gui/CommandExtensionPack.cpp

```diff
@@ -512,8 +512,8 @@ CmdTechDrawExtensionThreadHoleBottom::CmdTechDrawExtensionThreadHoleBottom()
 {
     sAppModule = "TechDraw";
     sGroup = QT_TR_NOOP("TechDraw");
-    sMenuText = QT_TR_NOOP("Add Cosmetic Thread Hole Bottom View");
-    sToolTipText = QT_TR_NOOP("Add a cosmetic thread to the top or bottom view of holes:<br>\
+    sMenuText = QT_TR_NOOP("Add cosmetic thread hole bottom view");
+    sToolTipText = QT_TR_NOOP("Adds a cosmetic thread to the top or bottom view of selected holes:<br>\
```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Adds a cosmetic thread to the top or bottom view of selected holes or circles")
```
then remove next 3 lines